sds_data_manager/stacks/sds_data_manager_stack.py

```diff
@@ -289,7 +239,7 @@ def __init__(
 
         # Adding Opensearch permissions
         indexer_lambda.add_to_role_policy(opensearch.opensearch_all_http_permissions)
-        # Adding s3 read permissions to get config.json
+        # Adding s3 read permissions
         indexer_lambda.add_to_role_policy(s3_read_policy)
         # Adding dynamodb write permissions
         indexer_lambda.add_to_role_policy(dynamodb_write_policy)
```

Comment:
Remove these? Since we aren't pulling config.json, or writing to dynamodb I don't think we need either of these permissions anymore.

Reply:
removed